log4j-core/src/main/java/org/apache/logging/log4j/core/layout/GelfLayout.java

```diff
@@ -135,14 +136,17 @@ public DeflaterOutputStream createDeflaterOutputStream(final OutputStream os) th
         @PluginBuilderAttribute
         private boolean includeThreadContext = true;
 
+        @PluginBuilderAttribute
+        private boolean includeTcpMessageDelimiter = false;
```

Comment:
Maybe name this as `includeNullMessageDelimiter` to indicate what it does, rather than what is can be used for?